app/uk/gov/hmrc/payetaxcalculatorfrontend/views/main_template.scala.html

```diff
@@ -23,6 +23,15 @@
     }
 }
 
+@scriptElems = {
+    <script src="@routes.Assets.at("javascripts/ga-events-emit.js")" type="text/javascript"></script>
+    @if(scriptElem.isDefined){@scriptElem}
+}
+
+@scriptElemAgg = {
```

Comment:
Isn't it like a constructor argument? scriptElem: Option[Html] = None
If yes then it is set outside and it can be None